ResearchKit/Common/ORKHelpers.m

```diff
@@ -395,6 +395,84 @@ CGFloat ORKTableViewLeftMargin(UITableView *tableView) {
     return font;
 }
 
+NSURL *ORKURLFromBookmarkData(NSData *data) {
+    if (data == nil) {
+        return nil;
+    }
+    
+    BOOL bookmarkIsStale = NO;
+    NSError *bookmarkError = nil;
+    NSURL *bookmarkURL = [NSURL URLByResolvingBookmarkData:data
+                                                   options:NSURLBookmarkResolutionWithoutUI
+                                             relativeToURL:nil
+                                       bookmarkDataIsStale:&bookmarkIsStale
+                                                     error:&bookmarkError];
+    if (! bookmarkURL) {
+        ORK_Log_Debug(@"Error loading URL from bookmark: %@", bookmarkError);
+    }
+    
+    return bookmarkURL;
+}
+
+NSData *ORKBookmarkDataFromURL(NSURL *url) {
+    if (! url) {
+        return nil;
+    }
+    
+    NSError *error = nil;
+    NSData *bookmark = [url bookmarkDataWithOptions:NSURLBookmarkCreationSuitableForBookmarkFile
+                     includingResourceValuesForKeys:nil
+                                      relativeToURL:nil
+                                              error:&error];
+    if (! bookmark) {
+        ORK_Log_Debug(@"Error converting URL to bookmark: %@", error);
+    }
+    return bookmark;
+}
+
+NSString *ORKPathRelativeToURL(NSURL *url, NSURL *baseURL) {
+    NSURL *url1 = [url URLByStandardizingPath];
+    NSURL *baseURL1 = [baseURL URLByStandardizingPath];
+    
+    NSString *path = [url1 absoluteString];
+    NSString *basePath = [baseURL1 absoluteString];
+    
+    if ([path hasPrefix:basePath]) {
+        NSString *relativePath = [path substringFromIndex:[basePath length]];
+        if ([relativePath hasPrefix:@"/"]) {
+            relativePath = [relativePath substringFromIndex:1];
+        }
+        return relativePath;
+    } else {
+        return path;
+    }
+}
+
+static NSURL *homeDirectory() {
```

Comment:
I tend to give static functions clearly non-extern naming, but it does look neater. Changing.